ext/openssl/openssl.c

```diff
@@ -115,6 +117,17 @@ enum php_openssl_cipher_type {
 	PHP_OPENSSL_CIPHER_DEFAULT = PHP_OPENSSL_CIPHER_RC2_40
 };
 
+/* add some encoding rules.  This is normally handled through filters
```

Comment:
NIT: If it's a sentence, should start with `A`

Reply:
corrected.